ecl/hql/hqlexpr.hpp

```diff
@@ -916,6 +916,7 @@ class HQL_API HqlParseContext
     bool expandCallsWhenBound;
     bool ignoreUnknownImport;
     bool aborting;
+    Linked<ICodegenContextCallback> codegenCtx;
```

Comment:
Trivial: It could be a pointer since I can't imagine its lifetime being shorter.  Keep it as it is though.